sass/themes/donate/donate-cr/components/header/_header.scss

```diff
@@ -19,8 +19,7 @@ header[role=banner] {
 }
 
 .block-branding {
-  margin: 0 auto;
-  
+

```

Comment:
umm

Reply:
Do we not need more, uh, code here?

Reply:
nope it works fine without in all browsers had it centred twice obviously IE11 didn't like it

Reply:
For some reason, I thought this PR was just removing a space and nothing else; my bad!